pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -1197,18 +1220,16 @@ func (fs *eosfs) GetMD(ctx context.Context, ref *provider.Reference, mdKeys []st
 		return nil, err
 	}
 
-	fn := ""
+	// TODO (lopresti) do we need this?
```

Comment:
This block can be removed as the path is not needed anymore for the auth of lw accounts

Reply:
This is what I thought too, and meant to ask you ;)